cluster/gce/gci/mounter/mounter

```diff
@@ -31,7 +31,7 @@ function gc {
     # Rkt pods end up creating new copies of mounts on the host. Hence it is ideal to clean them up right away.
     attempt=0
     until [ $attempt -ge 5 ]; do
-	${RKT_BINARY} gc --grace-period=0s &> /dev/null && break
```

Comment:
If the container is not GC, it causes those extra mounts which makes
1. mount table is very big if there are many mount points, which might be a performance concern since we list the mount points during unmount operations
2. Without this PR's fix, unmount device will fail because of those unrelated mount references. (umount device function first check whether there are other mount references for the mount path and will fail the operation if there are some exist)